spec/models/tag_prompt_deployment_spec.rb

```diff
@@ -8,6 +8,23 @@
   let(:question) {Question.new(questionnaire: questionnaire, type: 'tagging')}
   let(:questionnaire) {Questionnaire.new(id: 1)}
   let(:answer) {Answer.new}
+  let(:user1) {User.new(id: 1)}
+  let(:user2) {User.new(id: 2)}
+  let(:team_user1) {TeamsUser.new(user_id: user1.id, team_id: team.id)}
+  let(:team_user2) {TeamsUser.new(user_id: user2.id, team_id: team.id)}
+  let(:tagA) {AnswerTag.new(tag_prompt_deployment_id: tag_dep.id, user_id: user1.id, answer: answer, updated_at: Date.new.to_s)}
+  let(:tagB) {AnswerTag.new(tag_prompt_deployment_id: tag_dep.id, user_id: user2.id, answer: answer, updated_at: Date.new.to_s)}
+  answers = {
+    'answer1': {id: 1, question_id: 1, answer: 3, comments: 'comm', response_id: 2313},
+    'answer2': {id: 2, question_id: 1, answer: 3, comments: 'comment length exceeds threshold', response_id: 2313},
+    'answer3': {id: 3, question_id: 1, answer: 3, comments: 'comment length within threshold', response_id: 2313},
+    'answer4': {id: 4, question_id: 2, answer: 1, comments: 'com1', response_id: 241} }
+  answersObjectArray = answers.values.map do |answerParams| Answer.new(answerParams) end
+
+  short_comments_answers = {
```

Comment:
short_comments_answers  is not used anymore. Please retrain only the code that you're using. I have done a lot of changes to the new PR.